cmd/coder/urls.go

```diff
@@ -117,15 +147,16 @@ func (sub delSubCmd) Spec() cli.CommandSpec {
 	}
 }
 
-// devURLID returns the ID of a devURL, given the env name and port.
-// ("", false) is returned if no match is found.
-func devURLID(port string, urls []DevURL) (string, bool) {
+// devURLID returns the ID of a devURL, given the env name and port
+// from a list of DevURL records.
+// (false, "") is returned if no match is found.
+func devURLID(port int, urls []DevURL) (bool, string) {
```

Comment:
It's more idiomatic to return (string, bool).

Reply:
OK will swap.